app/controllers/users_controller.rb

```diff
@@ -49,6 +49,18 @@ def destroy
     redirect_to users_url
   end
 
+  def following
```

Comment:
RESTful là đường dẫn giúp lựa chọn control và action.
RESTful có các hàm là:created, read, updated, and deleted